### PR TITLE
Update of the BUIP and members page

### PR DESCRIPTION
**Pull request by torusJKL**
Added latest BUIPs and their status.
Removed members with memberships expired and added all members up to BUIP080.

### CODE REVIEW

**Comment by sickpig** [comment]
this is was so badly needed :P 
thanks going to review today

**Comment by torusJKL** [comment]
I have removed members based on the "Expired" flag of the voting system's
member list.
https://www.bitcoinunlimited.info/voting/render/member_list/17fedb2146b90528805cd8f43b585bfbc3db5e1343eb665d35e9976118a970ce

We might have to do another cleanup round based on the last votes on the
forum.

File: src/data/members.json [hunk 1]
```diff
@@ -58,20 +58,6 @@
             "Peter Tschipper is a full time Bitcoin Unlimited developer living on Vancouver Island, Canada. Peter holds a degree in Engineering from the University of Toronto and has worked in the areas of Finance, Health Care and Mineral Exploration. Most of his tech experiences are in the field of Performance Engineering and his main interest is in applying that knowledge to further the goal of Bitcoin On-Chain scaling."
         ]
     },
-    {
```
Comment (sickpig) [inline src/data/members.json]: @torusJKL the rules is that after a bunch of missing voting turns in a row the membership expired. I didn't check if the rule applies to all the account we are expiring here. so I wonder if @AndrewClifford could stamp this part of the PR, or at least if you have a resource curated by Andrew where he keeps track of the expired memberships

**Comment by AndrewClifford** [comment]
This is great to see @torusJKL. I wanted to see the result of the BUIP090. We have four more members   and it would be nice to append those as well, so we are finally up to date.
jcliff has also expired and can be removed.

**Comment by torusJKL** [comment]
@sickpig @AndrewClifford I have done the changes so that the member page should now be up to date.

**Pull request merged**